rpc/rpc-wrappers.go

```diff
@@ -78,6 +79,14 @@ type certificateRequest struct {
 	RegID int64
 }
 
+// ocspSigningRequest is a transfer object representing an OCSP Signing Request
```

Comment:
Because I didn't want the x509.Certificate object, which I later removed. Good catch, now it's naked enough!